### PR TITLE
list_of_events: Update push notifcation event

### PR DESCRIPTION
**Pull request by cmouse**
JIRA: DOV-3741

### CODE REVIEW

File: source/admin_manual/list_of_events.rst [hunk 1]
```diff
@@ -1242,13 +1242,20 @@ Finished the message submission.
...
+
+   ``mailbox``, ``mail_uid``, ``failed`` and ``push_notification_event`` were added.
+
++-------------------------+------------------------------------------------------+
+| Field                   | Description                                          |
++=========================+======================================================+
+| Inherits from mail user                                                        |
++-------------------------+------------------------------------------------------+
+| mailbox                 | Mailbox for event                                    |
+| mail_uid                | Mail uid (when present)                              |
```
Comment (jeffpc) [inline source/admin_manual/list_of_events.rst]: "When present"? Is there more detail we should provide about when it is/isn't present?

File: source/admin_manual/list_of_events.rst [hunk 2]
```diff
@@ -1242,13 +1242,20 @@ Finished the message submission.
...
+| failed                  | Set to ``yes`` on failure.                           |
```
Comment (jeffpc) [inline source/admin_manual/list_of_events.rst]: Hrm.  Should we use booleans instead?  I suppose the event code doesn't support them at the moment.

Comment (cmouse) [reply source/admin_manual/list_of_events.rst]: well, it's kinda of boolean. it's only there when it fails, so you can match it with `failed=*`

File: source/admin_manual/list_of_events.rst (hunk 1 above)
Comment (cmouse) [reply source/admin_manual/list_of_events.rst]: well, i suppose. it's there only when you have *mail* related event.

Comment (jeffpc) [reply source/admin_manual/list_of_events.rst]: Are there push notifications without a mail?

Comment (cmouse) [reply source/admin_manual/list_of_events.rst]: yes. mailbox creation/rename/delete/subscription events.

File: source/admin_manual/list_of_events.rst (hunk 2 above)
Comment (jeffpc) [reply source/admin_manual/list_of_events.rst]: Sure. I was thinking about explicit bool values - e.g., true/false in json.  Anyway, that's for another day.

File: source/admin_manual/list_of_events.rst (hunk 1 above)
Comment (jeffpc) [reply source/admin_manual/list_of_events.rst]: In that case, I'd suggest: s/when present/when the event is related to a specific mail/

Or something like that.

File: source/admin_manual/list_of_events.rst [hunk 3]
```diff
@@ -1242,13 +1242,20 @@ Finished the message submission.
...
+
++-------------------------+------------------------------------------------------+
+| Field                   | Description                                          |
++=========================+======================================================+
+| Inherits from mail user                                                        |
++-------------------------+------------------------------------------------------+
+| mailbox                 | Mailbox for event                                    |
+| mail_uid                | Mail uid (for mail related events)                   |
+| failed                  | Set to ``yes`` on failure.                           |
+| push_notification_event | Cause of push notification                           |
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst]: removed already.

**Pull request merged**